src/storage/admin/AdminProcessor.h

```diff
@@ -20,10 +20,21 @@ namespace storage {
 
 class TransLeaderProcessor : public BaseProcessor<cpp2::AdminExecResp> {
```

Comment:
Why `class` does not add comments?

Reply:
Thought it's easy to know these processor, so I skipped.
I'll add comments for classes to descrbe them.